src/Symfony/Component/Form/Tests/Extension/Core/DataMapper/PropertyPathMapperTest.php

```diff
@@ -113,6 +114,26 @@ public function testMapDataToFormsIgnoresUnmapped()
         $this->assertNull($form->getData());
     }
 
+    /**
+     * @requires PHP 7.4
+     */
+    public function testMapDataToFormsIgnoresUninitializedProperties()
+    {
+        $engineForm = new Form(new FormConfigBuilder('engine', \stdClass::class, $this->dispatcher));
+        $colorConfig = new FormConfigBuilder('color', \stdClass::class, $this->dispatcher);
+        $color = new \stdClass();
+        $colorConfig->setData($color);
+        $colorForm = new Form($colorConfig);
+
+        $car = new TypehintedPropertiesCar();
+        $car->engine = new \stdClass();
+
+        $this->mapper->mapDataToForms($car, [$engineForm, $colorForm]);
+
+        $this->assertSame($car->engine, $engineForm->getData());
+        $this->assertSame($color, $colorForm->getData());
```

Comment:
Could someone give me a hint why this fails on the Travis build? I'd like to fix it, but i can't reproduce the failed test for PHP 7.4.